index.html

```diff
@@ -1,19 +1,82 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
+    <header>
+      <img src="img/karma-logo.svg" alt="Logo for Karma company" />
+      <nav>
+        <ul>
+          <li class="meet-karma"><a href="#">Meet Karma</a</li>
+          <li><a href="#">How it works</a></li>
+          <li><a href="#">Store</a></li>
+          <li><a href="#">Blog</a></li>
+          <li><a href="#">Help</a></li>
+          <li><a href="#">Login</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <article class="hero">
+        <h1>Introducing Karma</h1>
+        <p>Bring WiFi with you, everywhere you go.</p>
+        <button>Learn More</button>
+      </article>
+
+      <article class="second-section">
+        <h1>Everyone needs a little Karma.</h1>
+        <div class="second-section-columns">
+          <div class="col-1">
+            <img src="img/icon-devices.svg" />
+            <p>Internet for all devices</p>
+          </div>
+          <div class="col-2">
+            <img src="img/icon-coffee.svg" />
+            <p>Boost your productivity</p>
+          </div>
+          <div class="col-3">
```

Comment:
You're not using these classes anywhere in your CSS, so you don't need them here. 
Additionally, each of these 3 blocks is basically exactly the same - so why use a different class? Classes are made to be re-usable. Always re-use code as much as possible!

Reply:
Thanks Lucy, unused classes have been removed